src/test/ui/proc-macro/keep-expr-tokens.stderr

```diff
@@ -1,15 +1,19 @@
 error[E0425]: cannot find function `missing_fn` in this scope
-  --> $DIR/keep-expr-tokens.rs:12:17
+  --> $DIR/keep-expr-tokens.rs:16:17
    |
 LL |     for item in missing_fn() {}
    |                 ^^^^^^^^^^ not found in this scope
 
 error[E0425]: cannot find value `bad` in this scope
-  --> $DIR/keep-expr-tokens.rs:14:62
+  --> $DIR/keep-expr-tokens.rs:18:62
    |
 LL |     (#[recollect_attr] #[recollect_attr] ((#[recollect_attr] bad)));
    |                                                              ^^^ not found in this scope
 
-error: aborting due to 2 previous errors
+error: language item required, but not found: `eh_personality`
+
+error: `#[panic_handler]` function required, but not found
```

Comment:
Could you avoid these errors by adding `extern crate std` (https://github.com/rust-lang/rust/pull/78296#discussion_r528338639)?